aten/src/ATen/CMakeLists.txt

```diff
@@ -250,6 +256,14 @@ if(USE_XPU)
   list(APPEND ATen_XPU_INCLUDE ${PROJECT_SOURCE_DIR}/third_party/ideep/mkl-dnn/include)
   list(APPEND ATen_XPU_INCLUDE ${XPU_MKLDNN_INCLUDE})
 
+  if(USE_KINETO)
+    find_package(Pti)
+    if(TARGET Pti::pti_view)
```

Comment:
Let's ask PTI to add a check flag in their cmake?

Reply:
ok

Reply:
do we have a JIRA to track this?
and, we should have a JIRA to check the mapping btw error code and string in PTI.

Reply:
The PTI is preinstalled. How to use PTI_NOT_FOUND?

`And they should provide a PTI version in their cmake, as well.`
will have a jira.

`we should have a JIRA to check the mapping btw error code and string in PTI.`
will have one.

Reply:
`we should have a JIRA to check the mapping btw error code and string in PTI.`
https://jira.devtools.intel.com/browse/PTI-87

`And they should provide a PTI version in their cmake, as well.`
What about they have a macro to define the version number in their header file `pti_view.h`?
We can use this macro to isolate code, for example, PTI <= 0.8 doesn't support translate the error code to msg.